example/topo/full_topo.go

```diff
@@ -24,17 +24,16 @@ func (t *FullTopology) GetLinkTypes() []string {
 	return []string{"Neighbors", "Master"}
 }
 
-func (t *FullTopology) GetNeighbors(linkType string, epoch uint64) []uint64 {
-	res := make([]uint64, 0)
-	switch {
-	case linkType == "Neighbors":
+func (t *FullTopology) GetNeighbors(linkType string, epoch uint64) (res []uint64) {
+	switch linkType {
+	case "Neighbors":
 		res = t.all
```

Comment:
Current implementation takes Task 0 as a worker and master.
It's Gang's decision.